homeassistant/components/tomorrowio/sensor.py

```diff
@@ -137,7 +131,6 @@ def convert_ppb_to_ugm3(molecular_weight: int | float) -> Callable[[float], floa
     TomorrowioSensorEntityDescription(
         key="global_horizontal_irradiance",
         attribute=TMRW_ATTR_SOLAR_GHI,
-        name="Global Horizontal Irradiance",
```

Comment:
Used a device class translation here. Please let me know if there is a significant change with irradiance and global horizontal irradiance